addons/mail/static/src/chatter/web/chatter.xml

```diff
@@ -28,6 +28,9 @@
                     <button class="btn btn-link text-action" t-att-class="{ 'o-active': state.isSearchOpen }" aria-label="Search Messages" title="Search Messages" t-on-click="onClickSearch">
                         <i class="oi oi-search" role="img"/>
                     </button>
+                    <button t-if="attachments.length" class="btn btn-link text-action" t-on-click="popoutAttachment">
```

Comment:
`attachments.length` => `state.thread.attachmentsInWebClientView`

This is almost the same, but the former take into account all attachments while the 2nd one only attachments that can be viewed